src/modules/core/components/map/map.tsx

```diff
@@ -0,0 +1,90 @@
+import { type FC, useState, useEffect, useRef } from 'react';
+import L, { divIcon } from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+import {
+  MapContainer,
+  TileLayer,
+  Popup,
+  Marker,
+  ZoomControl,
+} from 'react-leaflet';
+
+import { Zoom } from '../zoom/zoom';
+
+import styles from './map.module.scss';
+import avatar from './avatar.png';
+
+const customMarkerIcon = divIcon({
+  html: `<div class=${styles.custom_icon_container}>
+            <img class=${styles.custom_avatar} src=${avatar.src} alt='Marker' />
+          </div>`,
+  iconSize: [0, 0],
+  iconAnchor: [20, 50],
+  className: '',
+});
+
+const customMarker = divIcon({
+  html: `<div class=${styles.custom_marker_container}>
+            <div class=${styles.custom_marker_animation}></div>
+            <div class=${styles.custom_marker_center}></div>
+          </div>`,
+});
```

Comment:
rename to `viewerUserMarkerIcon` and move to `~/Project/src/modules/location/constants/map.constants.ts`